RELEASE/scripts/sl_ascend/sl_util.ash

```diff
@@ -2248,6 +2256,13 @@ boolean cloverUsageInit()
 		return true;
 	}
 
+	if(in_koe() && spleen_left() > 1 && canChew($item[lucky pill]) && item_amount($item[rare Meat isotope]) >= 50)
```

Comment:
This should probably be >= 40 instead of 50, or it'll sometimes say you have a clover when you don't.

Reply:
Made it >=60 and changed the other limit to isotopes/20 - 2, just so that we always have enough isotopes left to buy a Wand of Nagamar in hardcore.